lib/builder/webpack/base.js

```diff
@@ -101,19 +101,15 @@ export default class WebpackBaseConfig {
   }
 
   alias() {
+    const { srcDir, rootDir, dir: { assets: assetsDir, static: staticDir } } = this.options
+
     return {
-      '~': path.join(this.options.srcDir),
-      '~~': path.join(this.options.rootDir),
-      '@': path.join(this.options.srcDir),
-      '@@': path.join(this.options.rootDir),
-      [this.options.dir.assets]: path.join(
-        this.options.srcDir,
-        this.options.dir.assets
-      ),
-      [this.options.dir.static]: path.join(
-        this.options.srcDir,
-        this.options.dir.static
-      )
+      '~': path.join(srcDir),
+      '~~': path.join(rootDir),
+      '@': path.join(srcDir),
+      '@@': path.join(rootDir),
+      [assetsDir]: path.join(srcDir, assetsDir),
```

Comment:
I prefer the previous way of not de-structuring. It is somehow more clear to read.